tools/single_use/Engine_Change_MyISAM_to_INNODB_allTables.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Script changing all table engines in the database to InnoDB
```

Comment:
I think it would be a good idea to have the ability to do one table at a time. Instrument tables are going to be a pain if it's all or nothing.

Reply:
one table at a time could be done with a single SQL query.. I can try adding that